lib/mapper.js

```diff
@@ -15,39 +38,92 @@ var objCase = require('obj-case');
  * @api private
  */
 
-exports.track = function(track){
-  var props = track.properties();
-  if (track.revenue()) {
-    props.amount = track.revenue();
-    objCase.del(props, 'revenue');
-  }
-
+exports.track = function(track) {
   return {
     email: track.email(),
     userId: track.userId(),
     eventName: track.event(),
     createdAt: unixTime(track.timestamp()),
-    dataFields: props
+    dataFields: track.properties()
   };
 };
 
 /**
- * Map `identify`.
+ * Map `Added Product and Removed Product`.
  *
- * @param {Identify} track
+ * This is only invoked when a full products array is present
+ *
+ * @param {Track} track
  * @return {Object}
  * @api private
  */
 
-exports.identify = function(identify){
-  var traits = traverse(clone(identify.traits()));
-  var created = identify.created();
-  var userId = identify.userId();
-  objCase.del(traits, 'id')
-  if (created) traits.met = created.toISOString();
+exports.addedProduct =
+exports.removedProduct = function(track) {
+  var props = track.properties({ revenue: 'amount' });
```

Comment:
good catch — returns properties with `revenue` aliased to `amount`. originally i was passing it to `.dataFields` in this call but realized that wasn't called for in your implementation or the corresponding tests so I dropped that assignment. I'll remove it